### DISCUSSION TITLE
Enterprise Program API support

### DISCUSSION THREAD

**Discussion by barakwei** [AvdLee/appstoreconnect-swift-sdk#290] [General]
Apple has recently released the [Enterprise Program API](https://developer.apple.com/documentation/enterpriseprogramapi), which lets users automate tasks in their enterprise team. The API has its own OpenAPI spec, server etc, and it's probably 90% a subset of the App Store Connect API.
Do you have any thoughts about how the Enterprise Program API relates to this repo? Should it be in another package/repo or a part of this one? should it share the same objects?

**Comment by AvdLee** [comment]
I don't have any plans to add it myself, but if integrate well as a separate target in this package, I think it makes totally sense to add it to this repo!
